src/en/space-station-14/round-flow/proposals/ninja.md

```diff
@@ -0,0 +1,61 @@
+# Space Ninja V2
+
+| Designers | Implemented | GitHub Links |
+|---|---|---|
+| Warpzoned, Doru991 | :warning: Partially | TBD |
+
+## Overview
+
+This proposal is aimed towards addressing some issues that Space Ninjas currently possess, as well as adding some extra flavour and other miscellaneous content.</p>
+<p>New abilities have been added, and existing ones have been tailored to better fit with the stealth-based gameplay the Spider Clan so endorses. The Space Ninja now also come with an extra assortment of apparel, both useful and cool-looking.</p>
+<p>The Space Ninja's weapons have gotten a slight nerf, to compensate, new weapons and armament interactions have been added.
+
+## Background
+
+Currently, albeit being an extremely fun mid-round antag role and definitely one of the better ones, Space Ninjas aren't correctly geared towards what they should be the best at: Stealthing - Which is what this document hopes to address, mostly to add some more uniqueness to antagonist roles since a lot of them can be played by just killing everything in your way, and this includes Space Ninjas.
+
+## Spider Clan Code
+
+All Space Ninjas will now have a Spider Clan Code they will have to live by, and this thus affects gameplay substantially by imposing certain restrictions.
+- The already existing one, being the inability to shoot ranged weapons, will be included in this Code.
+- The second restriction is making it so the Space Ninjas can no longer tend to their own wounds - This includes applying topicals, injecting syringes, consuming pills, and drinking liquid chemicals, as it would be preferred that they are not even spotted during their mission. To compensate, Space Ninjas will now have increased passive regeneration as well as faster bloodloss recovery that far exceeds what normal crewmates could ever achieve due to their bio-engineered bodies.
```

Comment:
Will this ability be based on their suit power, turning the ninja into a "live and die on your suit and engineering" or will this be something seen on the actual ninja character?

Another consideration: can a ninja go into crit in a concealed spot and be slowly mended and revived given enough time? The idea is that if a ninja knows they might go crit to bloodloss/wounds soon they would find a spot to go into crit and hopefully come back out.

Reply:
> Will this ability be based on their suit power, turning the ninja into a "live and die on your suit and engineering" or will this be something seen on the actual ninja character?

This will be something on the character itself, the suit can only do so much.

> Another consideration: can a ninja go into crit in a concealed spot and be slowly mended and revived given enough time? The idea is that if a ninja knows they might go crit to bloodloss/wounds soon they would find a spot to go into crit and hopefully come back out.

I hadn't considered this, mostly due to the fact that from my point of view it'd be pretty much impossible for the ninja to go into crit from DoTs, although, if they do, yeah I can see them getting stabilizied again after a while.